Specs/DataSources/KmlDataSourceSpec.js

```diff
@@ -3022,10 +3026,176 @@ defineSuite([
                 expect(link2).toEqual(link2);
                 expect(folder2).not.toEqual(folder1);
                 expect(placemark2).not.toEqual(placemark1);
+
+                // Reset camera
+                options.camera.positionWC.x = 0.0;
```

Comment:
This is the only test that changes the camera. So I could put it there, but it won't do anything except after this test.